HTML/syntax_test_html.html

```diff
@@ -348,6 +348,14 @@
 ##          ^^ invalid.illegal.bad-comments-or-CDATA
 ##                      ^^^^^^^^^^^^^^^ - invalid.illegal.bad-comments-or-CDATA
 
+        <!-- 1st comment -- >
+##                       ^^^^ - punctuation
+        <p>Is a comment</p>
+##      ^^^^^^^^^^^^^^^^^^^ comment.block
+        <!-- 2nd comment -->
+##      ^^^^ invalid.illegal.bad-comments-or-CDATA
```

Comment:
Is this really illegal syntax in HTML5? I know that in XML, `--` is illegal inside of a comment unless it is part of the opening or closing token, but never heard of `<!` being illegal in comments.

Reply:
https://html.spec.whatwg.org/#comments

Reply:
This is what Firefox has to say about it ...

![grafik](https://user-images.githubusercontent.com/16542113/131900936-e99b8dc6-b7f1-444a-ab94-7661cb8ab237.png)